modules/vf-graphql-holochain/types.ts

```diff
@@ -1,6 +1,7 @@
 /**
  * base types for GraphQL query layer
  *
+ * 
  * @see https://github.com/valueflows/vf-graphql/blob/master/schemas/structs.gql
```

Comment:
@pospi I added the above line just so that I could ping you to ask about what this link should instead be? the one on line 5

Reply:
Looks old to me, I took it out in 8675452c 'cos the comment you have lower down in regard to the actual enum seems more useful.